src/templates/job/create.twig

```diff
@@ -15,8 +15,8 @@
 {% set elementType = 'craft\\elements\\Entry' %}
 {% set userElementType = 'craft\\elements\\User' %}
 
-{% set sources = craft.app.elementIndexes.getSources('craft\\elements\\Entry', 'index') %}
-{% set customizableSources = (sources is not empty and context == 'index' and currentUser.can('customizeSources')) %}
+{#{% set sources = craft.app.elementIndexes.getSources('craft\\elements\\Entry', 'index') %}#}
+{#{% set customizableSources = (sources is not empty and context == 'index' and currentUser.can('customizeSources')) %}#}
```

Comment:
why comment these out? same for `src/templates/job/edit.twig`

Reply:
Will remove this, since not used, thanks!